_src/pl/static/sass/_components.scss

```diff
@@ -436,6 +436,55 @@
     }
 }
 
+.pl-element-icons {
+
+    .example-set {
```

Comment:
Please see my [comments above about abstracting out example markup](https://github.com/edx/ux-pattern-library/pull/51/files#r28981009). I'd like this styling to be abstracted out as well and any special stuff between colors and this should still be here.